ioservice/ut/ios_buffer_pool.c

```diff
@@ -96,13 +89,13 @@ static int check_buffer_pool_per_domain(char *cs_argv[], int cs_argc, int nbp)
 	int rc;
 	int bp_count;
 	struct m0_rpc_server_ctx sctx = {
-		.rsx_xprts            = cs_xprts,
-		.rsx_xprts_nr         = ARRAY_SIZE(cs_xprts),
 		.rsx_argv             = cs_argv,
 		.rsx_argc             = cs_argc,
 		.rsx_log_file_name    = SERVER_LOG_FILE_NAME
 	};
 
+	sctx.rsx_xprts = m0_net_all_xprt_get();
```

Comment:
Actually, here, we can do the variable assignment at the original place.

But I think it is OK to have them updated here.